ceph_deploy/hosts/remotes.py

```diff
@@ -14,6 +14,15 @@ def platform_information(_linux_distribution=None):
     """ detect platform information from remote host """
     linux_distribution = _linux_distribution or platform.linux_distribution
     distro, release, codename = linux_distribution()
+    if not distro and os.path.isfile('/etc/os-release'):
```

Comment:
I think this should be considered the most canonical way of doing this, and the `if not distro` should be the fallback

Reply:
Just to clarify, we should check /etc/os-release *first*, then fallback to platform.linux_distribution() if that doesn't exist?

Reply:
yes, that sounds correct